### PR TITLE
Feature/responsive sidebar tweaked

### PR DESCRIPTION
**Pull request by shartrooper**
I had to create a new branch because I couldn't sync the newest commits to the former one, not even in my fork.
In any case, I made the requested changes.
image

### CODE REVIEW

File: components/layouts/AccountLayout.tsx [hunk 1]
```diff
@@ -1,20 +1,30 @@
+'use client'
```
Comment (devkiran) [inline components/layouts/AccountLayout.tsx:1]: You may remove this directive. We're using the pages router, so all are client components.

File: components/shared/Sidebar.tsx [hunk 2]
```diff
@@ -1,3 +1,5 @@
+'use client'
```
Comment (devkiran) [inline components/shared/Sidebar.tsx:1]: You may remove this directive. We're using the pages router, so all are client components.

**Review by devkiran — CHANGES_REQUESTED** [review]

File: components/layouts/AccountLayout.tsx (hunk 1 above)
Comment (shartrooper) [reply components/layouts/AccountLayout.tsx]: updated

**Comment by devkiran** [comment]
@shartrooper One issue i noticed during testing is that when sidebar is collapsed, the main content is freezed (buttons and inputs are not clickable). This makes app unusable when sidebar is collapsed. Can you check that please?

**Comment by shartrooper** [comment]
> @shartrooper One issue i noticed during testing is that when sidebar is collapsed, the main content is freezed (buttons and inputs are not clickable). This makes app unusable when sidebar is collapsed. Can you check that please?

this have to been caused by the event bubbling by clicking outside, ill check this out

**Comment by shartrooper** [comment]
@devkiran fixed. The issue was not removing the z-index stack value after collapsing. So it was overlapping even when not visible.

https://github.com/boxyhq/saas-starter-kit/assets/21326996/e9b578d3-f891-445b-b2b4-824474ff910f

**Review by devkiran — APPROVED** [review]

**Pull request merged**